src/test/java/com/vkatit/cinema/CinemaApplicationTests.java

```diff
@@ -14,9 +30,49 @@ class CinemaApplicationTests {
 	@Autowired
 	ApplicationContext context;
 
+	@MockBean
+	private Printable printable;
```

Comment:
mock is the worse way to test, please make proper integration tests.

Reply:
Mock removed
Integration test implemented